examples/query_lang/claims_alternatives.json

```diff
@@ -19,11 +19,10 @@
         {"id": "f", "path": ["email"]}
       ],
       "claim_sets": [ // defines the rules
```

Comment:
@tplooker I think you wanted to remove claim_sets? For a fair comparison, please modify all examples so that it is not being used any longer.